### PR TITLE
initial commit to apply edits

### PR DESCRIPTION
**Pull request by catlook**
making sure images display properly

### CODE REVIEW

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-bd3beb1fae/docs/rackspace-federation/).

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-088935215f/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-b464a2a03e/docs/rackspace-federation/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-0585a6780f/docs/rackspace-federation/). :bow:

File: doc/common/common-front.rst [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
+by using the industry standard
+`Security Assertion Markup Language (SAML) 2.0 <http://docs.oasis-open.org/security/saml/Post2.0/sstc-saml-tech-overview-2.0.html>`_
+protocol.
 
 |product name| is supported for users of Rackspace Cloud and Fanatical
-Support for AWS. Dedicated Hosting accounts are not currently
-supported, although the feature will be enhanced to include all customers
-in the future.
+Support for AWS. Although the feature will be enhanced to include all customers
+in the future, dedicated Hosting accounts are not currently supported.
```
Comment (rtgoodwin) [inline doc/common/common-front.rst]: "Dedicated" is a proper name here I believe.

File: doc/config-3p-saml/config-adfs/adfs-attribmapping.rst [hunk 2]
```diff
@@ -1,54 +1,53 @@
...
+The following example shows a Rackspace YAML (``.yml``) attribute mapping
+policy that you can use when you configure your identity provider with
+Rackspace. This example assumes that you have a group named
+``rackspace-billing`` with users that you want to access Rackspace billing
+services by using the ``billing:admin`` Rackspace role.
 
 Notes:
 
-- The ``groups`` specified in the example should be changed to match your
+- You should change the ``groups`` specified in the example match your
```
Comment (rtgoodwin) [inline doc/config-3p-saml/config-adfs/adfs-attribmapping.rst]: ...example to match...

File: doc/config-3p-saml/index.rst [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
-Configuring Third Party SAML Providers
-======================================
+====================================
+Configure third-Party SAML providers
+====================================
 
 This section provides information about configuring third-party SAML
 providers with |service|, including the setup considerations and
-information about Attribute Mapping.
+information about attribute mapping.
```
Comment (rtgoodwin) [inline doc/config-3p-saml/index.rst]: we use Attribute Mapping Policy elsewhere...I think I'm ok with this being lowercase, I think though. Just asking the question.

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-25ad2baec4/docs/rackspace-federation/).

**Comment by catlook** [comment]
squashing and merging!

**Pull request merged**